src/main/java/com/revature/services/EvalLogicImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.revature.services;
+
+import com.revature.repositories.EvalRepository;
+import com.revature.domain.Eval;
+
+import java.util.ArrayList;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Isolation;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly=false, isolation=Isolation.READ_COMMITTED)
+public class EvalLogicImpl implements EvalLogic {
+
+	@Autowired
+	private EvalRepository dao;
+
+	@Override
+	public Eval getEvalById(int id) {
+		Eval eval = dao.findOne(id);
+		System.out.println(eval);
+		return eval;
+	}
+	
+	@Override
+	public ArrayList<Eval> getEvalsByBatch(int id){
+		
+		ArrayList<Eval> evals = dao.findAllByBatchId(id);
+		System.out.println(evals);
+		return evals;
+	}
+	
+	@Override
+	public ArrayList<Eval> getEvalsByWeek(int id, int num){
+		ArrayList<Eval> evals = dao.findAllByWeek(id,num);
+		System.out.println(evals);
+		return evals;
+	}
+
+	@Override
+	public ArrayList<Eval> getEvalsByPerson(int id) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1319)